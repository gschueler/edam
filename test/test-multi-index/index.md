% Test Multi Index

multi docs with index