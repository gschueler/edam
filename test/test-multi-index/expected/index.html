<!DOCTYPE html>
<html>
<head>
  <title>Test Multi Index</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="pandoc" />
  <link rel="stylesheet" href="style.css" type="text/css" />
</head>
<body>


<nav class="page top">
    <ul>
        <li class="current"><a href="index.html">Test Multi Index</a></li>
        
        
        <li class="next"><a href="document.html">Chapter 1: Document</a></li>
    </ul>
</nav>
<div id="docbody">
<h1 class="title">Test Multi Index</h1>
<p
>multi docs with index</p
><h2 id="table-of-contents"
>Table of Contents</h2
><ol style="list-style-type: decimal;"
><li
  ><a href="document.html"
    >Document</a
    ></li
  ><li
  ><a href="document2.html"
    >Document2</a
    ></li
  ><li
  ><a href="document3.html"
    >Document3</a
    ></li
  ><li
  ><a href="document4.html"
    >Document4</a
    ></li
  ></ol
>
</div>

<nav class="page bottom">
    <ul>
        <li class="current"><a href="index.html">Test Multi Index</a></li>
        
        
        <li class="next"><a href="document.html">Chapter 1: Document</a></li>
    </ul>
</nav>

</body>
</html>